src/main/java/me/kodysimpson/simpapi/utility/TimeUtil.java

```diff
@@ -0,0 +1,181 @@
+package me.kodysimpson.simpapi.utility;
+
+/**
+ * Time Utility
+ *
+ * @author Negative
+ */
+public class TimeUtil {
+
+    public static String format(long l1, long l2) {
+        return format(l1, l2, false);
+    }
+
+    /**
+     * @param l1 - long 1
+     * @param l2 - long 2
+     * @param shortened - Is the time unit shortened?
+     * @return - Formatted message
+     *
+     * Example:
+     * format((System.currentTimeMillis() + 10000L), System.currentTimeMillis());
+     * would return 10 seconds, if shortened it would return 10s
+     */
+    public static String format(long l1, long l2, boolean shortened) {
```

Comment:
More descriptive names on the long arguments or even passing an Instant instance would probably be preferred here. You need to read the method's code to understand what they do.